migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,79 @@
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[discrete]
+[id='mtc-overview-upgrade-mtc']
+== Upgrade MTC
+xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[Upgrade MTC]: You can upgrade the MTC by using the Operator Lifecycle Manager.
+
+
+[discrete]
+[id='mtc-overview-mtc-checklists']
+== MTC checklists
+xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#premigration-checklists-mtc[MTC checklists]: Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), you can review the following checklists.
+
+* xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#cluster-health-checklist_premigration-checklists-mtc[Checklist for cluster health]
+* xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#source-cluster-checklist_premigration-checklists-mtc[Checklist for source cluster]
+* xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#target-cluster-checklist_premigration-checklists-mtc[Checklist for target cluster]
+
+[discrete]
+[id='mtc-overview-migrate-mtc-applications']
+== Migrate applications
+xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-with-mtc[Migrate applications]:  You can migrate your applications by using the Migration Toolkit for Containers (MTC) xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-mtc-web-console_migrating-applications-with-mtc[web console] or xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migrating-applications-cli_advanced-migration-options-mtc[the command line]. You can also xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migration-adding-cluster-to-cam_migrating-applications-with-mtc[add the cluster to the MTC web console].
+
+[discrete]
+[id='mtc-overview-advanced-migration-options']
+== Advanced migration options
+xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#advanced-migration-options-mtc[Advanced migration options]: You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-mtc[Create a registry route for direct image migration]
```

Comment:
Use a gerund for each bullet item so that they are grammatically parallel. Creating, Migrating, Configuring, etc.